modules/client/src/multisig/interfaces.ts

```diff
@@ -180,7 +180,7 @@ export type SubgraphMultisigApproversListItem = {
 };
 
 export type SubgraphMultisigVotingSettings = {
-  minApprovals: string; // TODO: Should be now a number?
+  minApprovals: string;
```

Comment:
Shouldn't this be a number?  Very low priority, can be skipped now

Reply:
Nope, subgraph returns a string, checked it

Reply:
I'll create a low priority task to fix that later